Concurrent  security (#43)
yiisoft/rate-limiter: README.md

### Implementing your own counter storage

> There are two ready to use stores counters available in the package:

Reviewer: ```suggestion
There are two ready to use counter storages available in the package:
```

> - `\Yiisoft\Yii\RateLimiter\Storage\SimpleCacheStorage` - to stores counters in any [PSR-16](https://www.php-fig.org/psr/psr-16/) cache.

Reviewer: ```suggestion
- `\Yiisoft\Yii\RateLimiter\Storage\SimpleCacheStorage` - stores counters in any [PSR-16](https://www.php-fig.org/psr/psr-16/) cache.
```

> - `\Yiisoft\Yii\RateLimiter\Storage\ApcuStorage` - to store counters under concurrent security by using the [APCu PHP extension](http://www.php.net/apcu).

Reviewer: ```suggestion
- `\Yiisoft\Yii\RateLimiter\Storage\ApcuStorage` - stores counters by using the [APCu PHP extension](http://www.php.net/apcu) while taking concurrency into account.
```

> To have your own storage implement `Yiisoft\Yii\RateLimiter\Storage\StorageInterface`. 

Reviewer: ```suggestion
To use your own storage implement `Yiisoft\Yii\RateLimiter\Storage\StorageInterface`. 
```
